homeassistant/components/tuya/cover.py

```diff
@@ -290,14 +289,41 @@ def open_cover(self, **kwargs: Any) -> None:
         value: bool | str = True
         if self.device.function[self.entity_description.key].type == "Enum":
             value = "open"
-        self._send_command([{"code": self.entity_description.key, "value": value}])
+        self._send_command(
+            [
+                {
+                    "code": self.entity_description.set_position,
```

Comment:
This is good (I think), but we should only set it when a `set_postion` DP has been defined).